src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -282,7 +309,6 @@ public void UseDevelopmentSettings()
         {
             Tracing.ConsoleLevel = TraceLevel.Verbose;
             Queues.MaxPollingInterval = TimeSpan.FromSeconds(2);
-            Singleton.ListenerLockPeriod = TimeSpan.FromSeconds(15);
```

Comment:
Yes - this lock period issue for local development (frequent restarts) has been very problematic.